pom.xml

```diff
@@ -1191,6 +1185,18 @@
                               <arg>-J--add-opens=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED</arg>
                               <arg>-J--add-opens=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED</arg>
                           </compilerArgs>
+                          <annotationProcessorPaths>
+                              <path>
+                                  <groupId>org.apache.activemq</groupId>
+                                  <artifactId>artemis-log-annotation-processor</artifactId>
+                                  <version>${project.version}</version>
+                              </path>
```

Comment:
hmmm... it will be difficult to maintain this down the road.

you don't specify log-annotation-processor if not using errorprone, but if using, then you do...


I won't know what's going in a few weeks from now.... a few months or an year for now and I have to debug anything this gets unpredictable, right?